check-eve.py

```diff
@@ -40,6 +40,23 @@ def validate_json(args, json_filename, schema):
     status = "OK"
     errors = []
 
+    with open(json_filename) as f:
+        flows = {}
+        for line in f:
+            obj = json.loads(line)
+            if obj["event_type"] == "flow" and "app_proto" in obj:
+                flows[obj["app_proto"]] = flows.get(obj["app_proto"], 0) + 1
+            if "stats" in obj:
+                d = obj["stats"]["app_layer"]["flow"]
+                stats = {}
+                for proto in d:
+                    stats[proto] = d[proto]
+                for k in flows:
+                    if k not in stats:
+                        errors.append("Different values for %s : %d flows and %d in stats" % (k, flows[k], 0))
+                    elif stats[k] != flows[k]:
+                        errors.append("Different values for %s : %d flows and %d in stats" % (k, flows[k], stats[k]))
```

Comment:
It looks like these are occurring, but never printed or cause failure.  Probably the "status" var has to be updated?

Might make sense to break out to its own function with a bit of function doc to says whats going on as well.

Reply:
Indeed